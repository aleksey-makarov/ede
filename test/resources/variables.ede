{ "string": "World", "integer": 1, "double": 3.25000, "bool": false, "map": { "key1": "value1", "key2": { "key3": "value2" } } }
---
Hello, {{ string }}!

{{ integer }} + {{ double }} > 5 is {{ bool }}

{{ map.key1 }}
{{ map.key2.key3 }}
