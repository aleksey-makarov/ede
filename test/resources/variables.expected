Hello, World!

1 + 3.25 > 5 is False

value1
value2
